webapp/resources/templates/fragments/pwdupdateform.html

```diff
@@ -7,7 +7,12 @@
 </script>
 
 <div class="alert alert-info">
-    <h2 th:utext="#{screen.mustchangepass.heading}"/>
+    <div th:if="${execInfo.processedTemplateName} == 'casExpiredPassView'">
```

Comment:
I would refactor this to a true/false format instead of matching a string.  That way if the template name ever changes, we don't have to update the name in multiple places.